go/private/actions/stdlib.bzl

```diff
@@ -44,13 +48,33 @@ def _stdlib_library_to_source(go, attr, source, merge):
         source["stdlib"] = _build_stdlib(go)
 
 def _should_use_sdk_stdlib(go):
-    return (go.mode.goos == go.sdk.goos and
+    return (_has_precompiled_stdlib(go.sdk.version) and
```

Comment:
Can we instead check the existence of `go.sdk.libs`? So people can download a pre-compile Go SDK even after 1.20 to avoid building it on every machine

Reply:
Done!